### ISSUE TITLE
Provide a glob pattern for only applying to specific files

### ISSUE THREAD

**Issue by ericmasiello** [jrdrg/eslint-plugin-sort-exports#36]
I'd love to use this but only apply it to so-called barrel files. These files are typically called `index.ts` or `index.tsx`. Is possible to add support for this?

```
{
    "sort-exports/sort-exports": [
        "error",
        { sortDir: "asc", ignoreCase: true, sortExportKindFirst: "type", include: ['src/**/index.{ts,tsx,js,jsx}'], }
    ]
}
```

**Comment by jrdrg** [comment]
Sure, I can try to add this

**Comment by ChildishForces** [comment]
#40 - This is a PR that solves this issue!

**Comment by jrdrg** [comment]
Thank you! I've merged and published v0.9.1
